src/test/scss/fozzie.spec.scss

```diff
@@ -0,0 +1,139 @@
+@use 'true' as *;
+@use 'src/scss/fozzie' as systemUnderTest;
+
+@include describe('Fozzie') {
+    @include it('does not throw any errors') {
```

Comment:
Would there be any additional value in potentially splitting out this test into subsections (multiple it calls).

May help when there is an error thrown just to put these into logical test sections

Reply:
I think that would be good. Might make the file a bit larger but the error will help pinpoint the thing that is failing plus would be easier to debug with X

Reply:
Yeah happy to - will give that a go now!